tox.ini

```diff
@@ -92,7 +99,8 @@ commands =
     # Install Synapse itself. This won't update any libraries.
     pip install -e ".[test]"
 
-    {envbindir}/coverage run "{envbindir}/trial"  {env:TRIAL_FLAGS:} {posargs:tests} {env:TOXSUFFIX:}
+    # run the regular trial command
+    {[testenv]commands}
```

Comment:
Looks like we're getting bit by tox-dev/tox#208, which should be fixed in tox-2.3.2:

`tox.ConfigError: ConfigError: No support for the posargs substitution type`

We could either update tox for this test, or re-duplicate the line from the default testenv